FusionIIIT/applications/counselling_cell/models.py

```diff
@@ -1,3 +1,61 @@
 from django.db import models
 
+from applications.academic_information.models import Student
 # Create your models here.
+
+
+POSTIONS= (
+    ('student_guide', 'Student_Guide'),
+    ('student_coordinator', 'Student_Coordinator'),
+)
+STATUS= (
+    ('status_unresolved', 'Unresolved'),
+    ('status_resolved', 'Resolved'),
+    ('status_inprogress', 'InProgress'),
+)
+
+class CounsellingInfo(models.Model):
+    student_id = models.ForeignKey(Student, on_delete=models.CASCADE)
+    student_position = models.CharField(max_length=50,choices=POSTIONS,default="pos1")
+
+    class Meta:
+        unique_together = (('student_id', 'student_position'),)
```

Comment:
This means that a student can have more than to positions at the same time. Is this the required use-case?